linux_os/guide/system/software/system-tools/package_gssproxy_removed/rule.yml

```diff
@@ -25,6 +25,11 @@ references:
 
 {{{ complete_ocil_entry_package(package="gssproxy") }}}
 
+{{% if product == "rhel8" %}}
+platforms:
+    - not_ovirt-host
```

Comment:
Something along: `This rule is disabled for Red Hat Virtualization, it will report not applicable. RHV uses NFS storage, which has dependency on gssproxy.`